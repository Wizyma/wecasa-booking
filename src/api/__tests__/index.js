import { getPrestations } from '../';

jest.mock('../index.js', () => ({
  getPrestations: jest.fn(() => Promise.resolve({ ressources: 'Coiffure' })),
}));

it('fetches data from unsplash', async () => {
  const data = await getPrestations();

  expect(data).toEqual({ ressources: 'Coiffure' });
  expect(data).toMatchSnapshot();
  expect(getPrestations).toHaveBeenCalledTimes(1);
});
