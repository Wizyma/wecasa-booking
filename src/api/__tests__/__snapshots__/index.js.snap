// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fetches data from unsplash 1`] = `
Object {
  "ressources": "Coiffure",
}
`;
